google-cloud-storage/src/test/java/com/google/cloud/storage/GrpcStorageImplUploadRetryTest.java

```diff
@@ -129,6 +132,44 @@ public void createFrom_inputStream() throws Exception {
     assertThat(service.returnError.get()).isFalse();
   }
 
+  @Test
+  public void startResumableWrite() throws Exception {
+
+    AtomicBoolean returnError = new AtomicBoolean(true);
+    StorageImplBase service =
+        new StorageImplBase() {
+          @Override
+          public void startResumableWrite(
+              StartResumableWriteRequest request, StreamObserver<StartResumableWriteResponse> obs) {
+            System.err.println("request = " + request);
+            if (request.equals(Resumable.startReq)) {
+              if (returnError.get()) {
+                returnError.compareAndSet(true, false);
```

Comment:
Please add a comment that this is used disable error after it occurs once.

Reply:
Will do